ocl/OCL.g4

```diff
@@ -199,7 +198,9 @@ factorExpression
 // ->subrange is used for ->substring and ->subSequence
 
 factor2Expression
-  : factor2Expression '->size()' 
+  : '-' factor2Expression 
+  | '+' factor2Expression  
```

Comment:
Do `'-'` and `'+'` have the same priority? I guess yes and it can be simplified:

```suggestion
  : ('-' | '+') factor2Expression 
```

Reply:
Yes, that would be ok to change like this.